README.md

```diff
@@ -1,97 +1,99 @@
-Sen samlar man på sig modet...och lovar sig själv att ta en lång renande dusch i någon snar framtid och kör:
-./replace_names.sh gammalt_namn nytt_namn
-Det här har gjorts några gånger redan, kolla här i repot:
-/create_bedfile/changes_made
-Men om man är rädd att man förstört något så finns en backup i samma mapp. 
+KK usually wants all the information they can get, allowing you to run the command without the `-l`flag.
 
-Historiskt sett, och inte egentligen på ett superorganiserat sätt så har jag haft min outputmapp för att skapa insilico-filer här:
-/medstore/Development/WGS_validation/in_silico_panels/
+### The genes/transcripts can’t be found
+Sometimes genes or transcripts can not be found in `refseq_20190301_ncbiRefSeq`. It could be that the transcript has been updated since 2019-03-01 or that the gene name has been changed. If an element of the input list was not found, the script will output a file named notfound.txt. Should the gene/transcript not be present in `refseq_20190301_ncbiRefSeq`there are a few things you can try to solve it:
 
-Men nu tänker jag att vi ska spara bedfilerna i det här repot, eftersom filerna ändå är ganska små och det möjliggör versionshantering! FANTASTISKT JU. 
+**Transcripts**
+Try deleting the version part of the transcript name in case there is another version present. If there is, check with the person requesting the panel update if it is okay to use the other version.
 
-För att vara lite bättre organiserad har jag sparat alla genlistor/transkriptlistor som har skickats i detta googlesheet. "InSilicopaneler KK" & "InSilicopaneler KG".
+**Genes names**
+Check if the gene has another name (e.g. by using [GeneCards database](https://www.genecards.org) ) that might be present in `refseq_20190301_ncbiRefSeq`. To replace a name use the script `replace_names.sh <old name> <new name>`. The changes will be saved in `changes_made`.  
+<br />
 
-# coverage analysis (for in silico panel verification into WGS analysis) 
+## The insilico\_panels folder
 
-you need to be root to run
+The in silico panels used in wopr are also stored here. Every panel folder contains a gene or transcript list as well as a bed file created by create\_bed.py. The lists and bed files of the panels currently used by wopr should also be in `WOPR/references/insilico_panels/`. 
+<br />
 
-cd to validate_wrapper subdirectory
+## Coverage analysis (for in silico panel verification into WGS analysis)
 
+In order to create a coverage analysis you will need a bed file for the panel in question.
+
+### How to run
+```
+# run as root
+cd validate_wrapper/
 ./validate_wrapper /path/to/bedfile.bed [departmentID = KG/KK,PAT] [annotationlevel = 1/2/3/4/5]
 
-results are saved here:
+# example:
+./validate_wrapper.sh /path/to/insilico_wgs/insilico_panels/KK/rubbningar_i_kopparmetabolism.v1.0/rubbningar_i_kopparmetabolism.v1.0.bed KK 1
+```
+### Result directory
+```
 /medstore/Development/WGS_validation/in_silico_panels/${department}/validate/panels/
```

Comment:
In the command above there is the option "PAT", but in the result directory there is only `klinisk_genetik` or `klinisk_kemi`. Do you know where the PAT results go?

Reply:
changed this validate_wrapper.sh :)

Reply:
Nice :)
The result folder does not exist for PAT though (`ls /medstore/Development/WGS_validation/in_silico_panels/`) so it could be that the code doesn't run because the first `mkdir` won't work. Perhaps you can test it with a fake panel?